### PR TITLE
Art locked stacks

### PR DESCRIPTION
**Pull request by lmcglohon**
Implements https://github.com/sul-dlss/solrmarc-sw/issues/101

### CODE REVIEW

File: stanford-sw/src/edu/stanford/Item.java [hunk 1]
```diff
@@ -124,6 +125,11 @@ public Item(DataField f999, String recId) {
 		else
 			hasLaneLoc = false;
 
+		if (StanfordIndexer.ART_LOCKED_LOCS.contains(itemType))
```
Comment (ndushay) [inline stanford-sw/src/edu/stanford/Item.java]: is this really looking for itemType?   vs currLoc and homeLoc?

Comment (lmcglohon) [inline stanford-sw/src/edu/stanford/Item.java]: yes, the locations are in the 999$t

Comment (ndushay) [inline stanford-sw/src/edu/stanford/Item.java]: ok, thanks.  Weird to me, but thanks for clarification - the data is the data.

File: stanford-sw/src/edu/stanford/StanfordIndexer.java [hunk 2]
```diff
@@ -1875,6 +1893,40 @@ private void setFundFacet(final Record record) {
...
+    }
+  }
+
+  List<VariableField> list852 = record.getVariableFields("852");
+  for (VariableField vf : list852)
+  {
+    DataField df = (DataField) vf;
+    String subC = MarcUtils.getSubfieldData(df, 'c');
+    if (ART_LOCKED_LOCS.contains(subC)){
+      locationFacet.add("Art Locked Stacks");
```
Comment (ndushay) [inline stanford-sw/src/edu/stanford/StanfordIndexer.java]: so what I'm wondering is if we want another enum class akin to stanford-sw/src/edu/stanford/enumValues/Genre.java  or if that is overkill.  I'm ok with punting on it for now - but maybe add a comment here to do it if we continue in this vein?

Comment (lmcglohon) [inline stanford-sw/src/edu/stanford/StanfordIndexer.java]: Maybe if we have more physical locations in the future?  I say we just punt for now.

Comment (ndushay) [inline stanford-sw/src/edu/stanford/StanfordIndexer.java]: works for me. mebbe I'll add a comment - looks like i have to tweak .travis.yml in any case.

**Comment by ndushay** [comment]
with tweaks to travis.yml  and  adding a comment to code, this is good to go once travis is happy.

**Pull request merged**